### PR TITLE
Compose: add escape hatch link

### PR DESCRIPTION
**Pull request by umbrae**
Compose is not yet ready on mweb, so we add a link to desktop compose as an alternative.

Fixes https://reddit.atlassian.net/browse/MOBILEWEB-634

Not very pretty, but it looks like this:

image

👓  @phil303

### CODE REVIEW

File: src/views/pages/messageCompose.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import BasePage from './BasePage';
 
 class MessageComposePage extends BasePage {
   render () {
+    const composeLink = "https://www.reddit.com/message/compose";
```
Comment (phil303) [inline src/views/pages/messageCompose.jsx]: Mostly a style nit, can you make this a constant outside of the class? Also note as part of the [js style guide](https://github.com/reddit/styleguide/tree/master/javascript#strings), we only use single quotes.

**Comment by phil303** [comment]
:fish: looks good but it's failing our linter on the single quote issue.

File: src/views/pages/messageCompose.jsx (hunk 1 above)
Comment (umbrae) [inline src/views/pages/messageCompose.jsx]: Hm I can, what's the thinking behind pulling it outside of the function just for learning purposes for me?

**Comment by umbrae** [comment]
:haircut: on those just for explicitness

File: src/views/pages/messageCompose.jsx (hunk 1 above)
Comment (phil303) [inline src/views/pages/messageCompose.jsx]: It's mostly stylistic as any performance gains are negligible. But if it's a constant variable that is "tweakable" and constant across function calls, I generally pull it out and put it at the top of a file. It's a slippery slope obviously (you could probably apply this thinking to the static text below for instance) so I'm never super gung-ho about it.

Also, sorry for not being explicit here, can you make it `COMPOSE_LINK`?

**Comment by phil303** [comment]
:nail_care:

File: src/views/pages/messageCompose.jsx (hunk 1 above)
Comment (schwers) [inline src/views/pages/messageCompose.jsx]: Upvote for the casing as `COMPOSE_LINK`, it's really nice to have it be obvious from the variable name that its a constant.

🔕  As a super minor nit, you can use the `reddit` property defined on `config` to build this variable, which would harden this for use in staging environments that have both mweb and desktop configured.

**Comment by umbrae** [comment]
💇  Ah, fixed those things, thanks. As an ES6 noob think I was mixing up `const` and `let`.

**Comment by phil303** [comment]
:fish:

**Comment by powerlanguage** [comment]
@umbrae - does this preserve the url params for `to` and `subject`?

E.g. https://m.reddit.com/message/compose?to=%2Fr%2Freddit.com

**Comment by umbrae** [comment]
As discussed verbally with @powerlanguage, it doesn't, but that's fine because it would be very difficult for someone to get to that URL on mobile now that we're unwhitelisting the compose resource. (They'd essentially have to be linked directly to m.reddit.com compose with those parameters, which is unlikely)

**Comment by schwers** [comment]
@umbrae is it safe to close this PR now?

**Comment by umbrae** [comment]
Yes, thanks

**Pull request closed without merging**